pyaedt/hfss3dlayout.py

```diff
@@ -548,33 +573,32 @@ def create_frequency_sweep(self, setupname, unit, freqstart, freqstop, num_of_fr
         setupname : str
             Name of the setup that is attached to the sweep.
         unit :
-            Units such as ``"MHz"``, ``"GHz"``, and so on.
-        freqstart :
+            Units, such as ``"MHz"`` or ``"GHz"``.
+        freqstart : float
             Starting frequency of the sweep.
-        freqstop :
+        freqstop : float
             Stopping frequency of the sweep.
+        num_of_freq_points : int
+            Number of frequency points in the range.
         sweepname : str
             Name of the sweep. The default is ``None``.
-        num_of_freq_points :
-            Number of frequency points in the range.
-        sweeptype : discrete
-            Type of sweep. Choices are ``"Fast"``, ``"Interpolating"``, and ``"Discrete"``. The default is ``"Interpolating``.
-        interpolation_max_solutions :
-            Maximum number of solutions evaluated for the interpolation process. The default is ``250``.
-        interpolation_tol_percent :
+        sweeptype : str, optional
+            Type of the sweep. Options are ``"Fast"``, ``"Interpolating"``, and ``"Discrete"``. The default is ``"Interpolating``.
+        interpolation_tol_percent : float, optional
             Error tolerance threshold for the interpolation process. The default is ``0.5``.
-        save_fields : bool
-            Save the fields for a discrete sweep only. The default is ``True``.
-        save_rad_fields_only : bool
-            Save only the radiated fields if ``save_fields = True``. The default is ``False``.
-        use_q3d_for_dc : bool
-            Use Q3D to solve the DC point. The default is ``False``.
+        interpolation_max_solutions : int, optional
+            Maximum number of solutions evaluated for the interpolation process. The default is ``250``.
+        save_fields : bool, optional
+            Whether to save the fields for a discrete sweep only. The default is ``True``.
+        save_rad_fields_only : bool, optional
+            Whether to save only the radiated fields if ``save_fields = True``. The default is ``False``.
```

Comment:
```suggestion
            Whether to save only the radiated fields if ``save_fields=True``. The default is ``False``.
```

No spaces for input parameters.